### PR TITLE
[ACS-563] update script to remove only identity namespace from terraform.tfstate

### PR DESCRIPTION
**Pull request by SohelSaiyed**
- add a dependency for ALB to wait for nginx-ingress chart to be deployed, this is to make autoscaling group have healthy instance before they are mounted to ALB target group.

### CODE REVIEW

File: modules/acs/main.tf [hunk 1]
```diff
@@ -6,7 +6,7 @@ module "acs_admin_password_md4" {
     ACS_ADMIN_PASSWORD = var.create_acs ? var.acs_admin_password_secret : "admin"
   }
 
-  command = "printf %s \"$ACS_ADMIN_PASSWORD\" | iconv -t UTF-16LE | openssl md4 | awk '{print $2}'"
+  command = "printf %s \"$ACS_ADMIN_PASSWORD\" | iconv -t UTF-16LE | openssl md4 | awk '{print $1}'"
```
Comment (mteodori) [inline modules/acs/main.tf:9]: is this change required?

Comment (SohelSaiyed) [reply modules/acs/main.tf:9]: something I found $1 works with most of Linux and mac version and not with $2

Comment (mteodori) [reply modules/acs/main.tf:9]: so not working only for @igdianov on the windows setup I believe, let's remove and create a different PR maybe

**Review by mteodori — APPROVED** [review]
